### DISCUSSION TITLE
Netbox does not display navigation tabs on the left side of the browser

### DISCUSSION THREAD

**Discussion by wypxl** [netbox-community/netbox#14306] [General]
Hi，guys
My NetBox version is 3.6.0, my python version is 3.9.5.

I found that Netbox couldn't display the left side navigation TAB bar on most versions of the browser, i wonder if I can fix this problem by changing one of the Netbox parameters instead of upgrading the browser version. Have you ever met this issue?

**Comment by candlerb** [comment]
Can you show a screenshot? And list which specific browser versions work and don't work? Have you tried hovering over or clicking on the Netbox logo in the top left hand corner? Have you opened your browser's developer console and looked for errors?

There have been cases before of ancient browsers not working because they are missing support for newer Javascript features. In general, the solution is to upgrade the browser, although sometimes Netbox's Javascript is modified to work with older browsers (if you can point to the exact problem and someone wants to volunteer to make the change).

See for example #12974, #8331.

In some cases it could be a problem with your apache/nginx proxy setup, e.g. you are using https:// to access the page but have not set X-Forwarded-Proto correctly, then Netbox may generate http:// links which are rejected by the browser as "mixed content". But this usually only affects rack elevations.
